### DISCUSSION TITLE
显示重复注册被封禁

### DISCUSSION THREAD

**Discussion by abcdlsj** [BurnHair/docs#125] [技术支持]
用的 GitHub 账号注册，确定没有重复注册，是还有其它判断吗
```
{
  "error": {
    "message": "用户7889已被封禁：重复注册 会话重复 (request id: 20240626061926801867706KRwgTzny) (request id: 2024062606192611960711714051994)",
    "type": "one_api_error",
    "param": "",
    "code": null
  }
}
```

**Comment by BennyThink** [comment]
你是不是和朋友共享 你的bearer token，然后挂到同一个one-api去用？ 仅此一次，请勿共享 API Key

**Reply by abcdlsj** [reply]
没有共享，是根据请求 IP 判断的么，我这边会在多个电脑上使用

现在是挂到用 render 搭建的 one-api 使用的，不知道这个 IP 会不会变

**Reply by BennyThink** [reply]
你怎么和这个人回复一模一样 https://github.com/orgs/BurnHair/discussions/124#discussioncomment-9880383

尽量避免此类操作，很容易被判定为重复注册。现在重复注册很多，而且在NSFW，为了稳定性，我也很没办法哪怕错杀也没太多办法。

**Reply by abcdlsj** [reply]
ok, thx
